profile/README.md

```diff
@@ -3,7 +3,7 @@ Intelligent Machine Learning (IML) targets to set up a full-stack, high-performa
 
 ## Latest News
 
-- [2025/01] [EDiT: A Local-SGD-Based Efficient Distributed Training Method for Large Language Models, ICLR'24.](https://arxiv.org/abs/2412.07210)
+- [2025/01] [EDiT: A Local-SGD-Based Efficient Distributed Training Method for Large Language Models, ICLR'25.](https://arxiv.org/abs/2412.07210)
```

Comment:
update to your project md in atorch？

Reply:
Done. Please refer to https://github.com/intelligent-machine-learning/atorch/pull/6.